### ISSUE TITLE
Buffer ERR_INVALID_ARG_TYPE error

### ISSUE THREAD

**Issue by olets** [ericmorand/gulp-twing#26]
Back looking at porting some gulpfiles over to use twing. Maybe this is the time it'll stick!

Getting a type error with the readme's minimal example.

```json
(in package.json, after `yarn add gulp gulp-twing twing`)

  "devDependencies": {
    "gulp": "^4.0.2",
    "gulp-twing": "^3.0.1",
    "twing": "^4.0.2"
  }
```

```twig
{# src/index.twig, but doesn't matter - I get the error whether or not there are any files for the gulp.src #}

{{ 'hello world' }}
```

```javascript
// gulpfile.js

const gulp = require('gulp')
const twing = require('gulp-twing')
const {TwingEnvironment, TwingLoaderRelativeFilesystem} = require('twing')
let env = new TwingEnvironment(new TwingLoaderRelativeFilesystem())

function twig() {
  return gulp.src('**/*.twig')
    .pipe(twing(env))
    .pipe(gulp.dest('dist'))
}

exports.default = twig
```

```shell
gulp
Using gulpfile path/to/gulpfile.js
Starting 'default'...
'default' errored after 386 ms
TypeError [ERR_INVALID_ARG_TYPE]: The first argument must be one of type string, Buffer, ArrayBuffer, Array, or Array-like Object. Received type object
    at Function.from (buffer.js:305:9)
    at Transform.stream._transform (path/to/node_modules/gulp-twing/src/index.js:35:36)
    at Transform._read (_stream_transform.js:189:10)
    at Transform._write (_stream_transform.js:177:12)
    at doWrite (_stream_writable.js:435:12)
    at writeOrBuffer (_stream_writable.js:419:5)
    at Transform.Writable.write (_stream_writable.js:309:11)
    at DestroyableTransform.ondata (path/to/node_modules/readable-stream/lib/_stream_readable.js:619:20)
    at DestroyableTransform.emit (events.js:223:5)
    at DestroyableTransform.EventEmitter.emit (domain.js:498:23)
```

**Comment by jasjc** [comment]
I'm also having the same issue with the suggested bare bones setup in Gulp

**Comment by pingoo** [comment]
same here :(

**Comment by azt3k** [comment]
adding this to the package.json worked for me:

`"twing": "^2.1.2"`

**Comment by dnnsjsk** [comment]
@azt3k solution is indeed solving the problem, but I wish to use the newest version of twing. 

Any other ideas?

**Comment by e-r-n-i-e** [comment]
> adding this to the package.json worked for me:
> 
> `"twing": "^2.1.2"`

It looks like the highest twing major version that is working with gulp-twing is 3. I added this to my package.json and it worked for me:
`    "twing": "^3"`

**Comment by ericmorand** [comment]
Hello all,

Sorry for the long silence about this issue. My personal life has been extremely crowded these last few months (for the best) but I now have times to take care of twing and its side-projects again.

Looking at this issue right now...

**Comment by ericmorand** [comment]
This should be fixed in the upcoming 4.0.0 release. This is obviously a breaking change since Twing is entirely asynchronous since version 4 - which is exactly why gulp-twing wasn't functioning properly with anything higher than Twing@3.

I did set the peerDependency to at least Twing@5.0.2. Twing@4 is not maintained anymore.

**Comment by e-r-n-i-e** [comment]
Thanks a lot, works fine!
